apps/dashboard/src/app/nebula-app/(app)/components/CustomChat/CustomChatContent.tsx

```
@@ -0,0 +1,345 @@
+        const textMessage = messageToSend.content.find(
+          (x) => x.type === "text",
+        );
+        if (textMessage) {
+          textMessage.text = `${messagePrefix}\n\n${textMessage.text}`;
+        }
+      }
+
+      try {
+        setChatAbortController(abortController);
+        // --- Custom API call ---
+        const payload: any = {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)